build_cli/pubspec.yaml

```diff
@@ -30,3 +30,7 @@ dev_dependencies:
   path: ^1.0.0
   source_gen_test: ^1.0.0
   test: ^1.0.0
+
```

Comment:
Change the `build_cli_annotations` constraint above to `'>=2.1.0 <2.2.0'`

Also change the version of `build_cli` to 2.2.0 – also add a changelog entry